fe/fe-core/src/main/java/com/starrocks/catalog/EsTable.java

```diff
@@ -202,6 +203,11 @@ private void validate(Map<String, String> properties) throws DdlException {
         }
         hosts = properties.get(KEY_HOSTS).trim();
         seeds = hosts.split(",");
+        for (String seed : seeds) {
```

Comment:
Seed should trim? Like "http://1.1.1.1; http://1.1.1.1" the second seed has a space

Reply:
yes, better to trim again for each seed